packages/agent/src/canisterStatus/index.ts

```diff
@@ -211,6 +225,8 @@ export const request = async (options: {
           }
         }
       } catch (error) {
+        error;
+        console.log(error);
```

Comment:
leftover from testing?

Reply:
Oh wait, this had changes from another feature left in it, one sec